nuget/NetAnalyzers/Microsoft.CodeAnalysis.NetAnalyzers.Package.csproj

```diff
@@ -24,7 +24,19 @@
     <AnalyzerNupkgAssembly Include="Microsoft.CodeAnalysis.CSharp.NetAnalyzers.dll" />
     <AnalyzerNupkgAssembly Include="Microsoft.CodeAnalysis.VisualBasic.NetAnalyzers.dll" />
 	</ItemGroup>
-  
+
+  <ItemGroup>
+    <AnalyzerLegacyRuleset Include="LegacyRulesets\AllRules.ruleset" />
```

Comment:
@dotpaul I have moved all the shipping legacy rulesets, including the `SecurityRules.ruleset` to `Microsoft.CodeAnalysis.NetAnalyzers` package. Let me know if you feel we no longer need to ship these.

Reply:
`SecurityRules.ruleset` doesn't look maintained, so I'd just remove it. (Do we need any of the legacy rulesets?)

Reply:
> Do we need any of the legacy rulesets?

I don't think so. Happy to remove all of them :-)

Reply:
Done with https://github.com/dotnet/roslyn-analyzers/pull/4504/commits/8064abdb46af1f9bf76b2d142cc1a390bc96632f